homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -35,6 +35,8 @@
 
 REGIONS = ['US', 'NA', 'EU', 'IL']
 
+TRACKABLE_DOMAINS = ['sensor', 'input_text']
+
 SCAN_INTERVAL = timedelta(minutes=5)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
```

Comment:
Needs to be updated as now are also `entity_id`s allowed for `origin` and `destination`.